### PR TITLE
Automate several Ascended Celestial archetype features 

### PR DESCRIPTION
**Pull request by pedrogrullada**
- Remove unused tags
- Implement Ascended Celestial nimbus light as an aura
- Automate Celestial Rebirth's traits, immunities, and spell alteration
- Add mergeable roll option and alteration for Cleansing Spell
- Add truesight sense and perception bonus to Eyes Unclouded
- Automate Scouring Spark (double damage implemented as Note, not possible otherwise at the moment)
- Add effects for Abjure Harm, Bless Ally, Aegis for the Innocent, Ascend, Celestial Nimbus's fear bonus, Celestial Armaments, Channel Divine Spark and Shining Glory.

### CODE REVIEW

File: packs/feat-effects/effect-ascend.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "_id": "5rypwpGjbu4aCAHh",
+    "img": "icons/magic/air/weather-wind-gust.webp",
+    "name": "Effect: Ascend",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Ascend]</p>\n<p>Your fly speed becomes equal to double your Speed for 5 rounds.</p>"
```
Comment (TikaelSol) [inline packs/feat-effects/effect-ascend.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Ascend]</p>\n<p>Your fly speed becomes equal to double your Speed.</p>"
```

File: packs/feat-effects/effect-celestial-armaments-allies.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        "rules": [
+            {
+                "choices": {
+                    "ownedItems": true,
+                    "types": [
+                        "weapon"
+                    ]
+                },
+                "flag": "weapon",
+                "key": "ChoiceSet"
```
Comment (simonward) [inline packs/feat-effects/effect-celestial-armaments-allies.json:32]:
```suggestion
                "key": "ChoiceSet",
                "prompt": "PF2E.SpecificRule.Prompt.Weapon"
```

File: packs/feat-effects/effect-celestial-armaments.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            {
+                "choices": {
+                    "includeHandwraps": true,
+                    "ownedItems": true,
+                    "types": [
+                        "weapon"
+                    ]
+                },
+                "flag": "weapon",
+                "key": "ChoiceSet"
```
Comment (simonward) [inline packs/feat-effects/effect-celestial-armaments.json:33]:
```suggestion
                "key": "ChoiceSet",
                "prompt": "PF2E.SpecificRule.Prompt.Weapon"
```

File: packs/feat-effects/effect-celestial-armaments.json [hunk 4]
```diff
@@ -0,0 +1,86 @@
+{
+    "_id": "h8BLaHUFnzCDJf5I",
+    "img": "icons/skills/melee/weapons-crossed-swords-yellow-teal.webp",
+    "name": "Effect: Celestial Armaments",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Celestial Armaments]</p>\n<p>Select one weapon or handwraps of mighty blows. The weapon sheds light like a torch, deals an additional 1d8 spirit damage, and you gain the weapon's critical specialization.</p>\n<p>When you critically hit a target with the weapon, the target must succeed at a Fortitude save against your class DC or spell DC (whichever is higher) or be blinded for 1d4 rounds.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-celestial-armaments.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Celestial Armaments]</p>\n<p>Select one weapon or <em>handwraps of mighty blows</em>. The weapon sheds light like a torch, deals an additional 1d8 spirit damage, and you gain the weapon's critical specialization.</p>\n<p>When you critically hit a target with the weapon, the target must succeed at a Fortitude save against your class DC or spell DC (whichever is higher) or be blinded for 1d4 rounds.</p>"
```

File: packs/feats/a-thousand-cries-for-help.json [hunk 5]
```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You've learned to communicate with creatures from all over the world, overcoming language barriers to lend a hand and help where you're needed. You're affected by a constant truespeech spell.</p>\n<p>When you attempt a Diplomacy check to @UUID[Compendium.pf2e.actionspf2e.Item.Make an Impression] or Request, you can spend 1 Mythic Point to make the check at mythic proficiency.</p>"
+            "value": "<p>You've learned to communicate with creatures from all over the world, overcoming language barriers to lend a hand and help where you're needed. You're affected by a constant truespeech spell.</p>\n<p>When you attempt a Diplomacy check to @UUID[Compendium.pf2e.actionspf2e.Item.Make an Impression] or @UUID[Compendium.pf2e.actionspf2e.Item.Request], you can spend 1 Mythic Point to make the check at mythic proficiency.</p>"
```
Comment (simonward) [inline packs/feats/a-thousand-cries-for-help.json:14]:
```suggestion
            "value": "<p>You've learned to communicate with creatures from all over the world, overcoming language barriers to lend a hand and help where you're needed. You're affected by a constant @UUID[Compendium.pf2e.spells-srd.Item.Truespeech] spell.</p>\n<p>When you attempt a Diplomacy check to @UUID[Compendium.pf2e.actionspf2e.Item.Make an Impression] or @UUID[Compendium.pf2e.actionspf2e.Item.Request], you can spend 1 Mythic Point to make the check at mythic proficiency.</p>"
```

File: packs/feat-effects/effect-celestial-armaments.json [hunk 6]
```diff
@@ -0,0 +1,87 @@
+{
+    "_id": "h8BLaHUFnzCDJf5I",
+    "img": "icons/skills/melee/weapons-crossed-swords-yellow-teal.webp",
+    "name": "Effect: Celestial Armaments",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Celestial Armaments]</p>\n<p>Select one weapon or <em>handwraps of mighty blows</em>. The weapon sheds light like a torch, deals an additional 1d8 spirit damage, and you gain the weapon's critical specialization.</p>\n<p>When you critically hit a target with the weapon, the target must succeed at a Fortitude save against your class DC or spell DC (whichever is higher) or be blinded for 1d4 rounds.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-celestial-armaments.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Celestial Armaments]</p>\n<p>Your weapon sheds light like a torch, deals an additional 1d8 spirit damage, and you gain the weapon's critical specialization.</p>\n<p>When you critically hit a target with the weapon, the target must succeed at a Fortitude save against your class DC or spell DC (whichever is higher) or be blinded for 1d4 rounds.</p>"
```

File: packs/feat-effects/effect-aegis-for-the-innocent.json [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+        },
+        "level": {
+            "value": 14
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder War of Immortals"
+        },
+        "rules": [
```
Comment (simonward) [inline packs/feat-effects/effect-aegis-for-the-innocent.json:23]: Do we want to add a 10 foot aura RE?

File: packs/feats/eyes-unclouded.json [hunk 8]
```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>Your senses are enhanced by your divine spark, enabling you see things as they truly are, piercing through illusions, transformations, and lies. You're affected by a constant truesight spell. Additionally, you gain a +2 circumstance bonus to Perception checks to @UUID[Compendium.pf2e.actionspf2e.Item.Sense Motive] and to Perception DCs against attempts to Lie to you. When a creature attempts to Lie to you, they roll the Deception check to Lie twice and use the worse result; this is a misfortune effect.</p>"
+            "value": "<p>Your senses are enhanced by your divine spark, enabling you to see things as they truly are, piercing through illusions, transformations, and lies. You're affected by a constant @UUID[Compendium.pf2e.spells-srd.Item.Truesight] spell. Additionally, you gain a +2 circumstance bonus to Perception checks to @UUID[Compendium.pf2e.actionspf2e.Item.Sense Motive] and to Perception DCs against attempts to Lie to you. When a creature attempts to Lie to you, they roll the Deception check to Lie twice and use the worse result; this is a misfortune effect.</p>"
```
Comment (simonward) [inline packs/feats/eyes-unclouded.json:14]:
```suggestion
            "value": "<p>Your senses are enhanced by your divine spark, enabling you to see things as they truly are, piercing through illusions, transformations, and lies. You're affected by a constant @UUID[Compendium.pf2e.spells-srd.Item.Truesight] spell. Additionally, you gain a +2 circumstance bonus to Perception checks to @UUID[Compendium.pf2e.actionspf2e.Item.Sense Motive] and to Perception DCs against attempts to @UUID[Compendium.pf2e.actionspf2e.Item.Lie] to you. When a creature attempts to Lie to you, they roll the Deception check to Lie twice and use the worse result; this is a misfortune effect.</p>"
```

Comment (simonward) [reply packs/feats/eyes-unclouded.json:14]: Might be able to use an EphemeralEffect for the roll twice on attempts to Lie to you.

File: packs/feat-effects/effect-aegis-for-the-innocent.json (hunk 7 above)
Comment (pedrogrullada) [reply packs/feat-effects/effect-aegis-for-the-innocent.json:23]: It doesn't say it's an aura, so I held off

File: packs/feats/eyes-unclouded.json (hunk 8 above)
Comment (pedrogrullada) [reply packs/feats/eyes-unclouded.json:14]: Yeah, I held off on that as I couldn't get it to work, there may be a bug. I reported it on the bugs channel a few days ago.

**Pull request merged**